python/ray/log_monitor.py

```diff
@@ -361,4 +361,5 @@ def run(self):
                    f"failed with the following error:\n{traceback_str}")
         ray.utils.push_error_to_driver_through_redis(
             redis_client, ray_constants.LOG_MONITOR_DIED_ERROR, message)
+        logger.error(message)
```

Comment:
Does this get double-logged?

Reply:
No. Previously all exceptions were logged because we redirected the stderr to the file. Now we don't do that anymore and just use the logger. So it won't be logged twice.